private/buf/cmd/buf/command/config/configlslintrules/configlslintrules.go

```diff
@@ -121,7 +121,10 @@ func run(
 			}
 		}
 	} else {
-		rules = config.Lint.GetRules()
+		rules, err = config.Lint.GetRules()
+		if err != nil {
+			return err
+		}
```

Comment:
Yeah, I think I like this idea -- moving it up to `buflint` and `bufbreaking` respectively is also good, because the `internal` package is actually internal to `bufcheck`, and not the config packages. Going to update these + the naming :D